README.md

```diff
@@ -1,18 +0,0 @@
-## Rust "Simple Schnorr Multi-Signatures"
```

Comment:
Why are `README.md` files are removed? I think those are pretty important files for people to get info about project.